sycl/test-e2e/KernelFusion/sync_acc_mem_op.cpp

```diff
@@ -1,9 +1,6 @@
 // RUN: %{build} %{embed-ir} -o %t.out
 // RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s
 
-// Windows doesn't yet have full shutdown().
-// UNSUPPORTED: ze_debug && windows
```

Comment:
As kernel fusion is currently unsupported on Windows, all e2e tests for kernel fusion are marked as unsupported on Windows here: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/KernelFusion/lit.local.cfg#L7

So I'm not sure this change and the change to the other files in this directory are actually required.

Reply:
> As kernel fusion is currently unsupported on Windows, all e2e tests for kernel fusion are marked as unsupported on Windows here: https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/KernelFusion/lit.local.cfg#L7
> 
> So I'm not sure this change and the change to the other files in this directory are actually required.

I think it's all the more reason to remove it! The line gives the false impression that we would run it on Windows with ZE_DEBUG if available.

Reply:
Sorry, brain fart, I thought you added the `UNSUPPORTED` line, not removed it. 

You're of course completely right.